<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="QA-01" failures="1" tests="1" name="Test" time="0.027" errors="0" timestamp="2020-03-03T13:53:42 EET">
  <testcase classname="com.w2a.rough.TestNG" name="test2" time="0.007">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [2] but found [1],
	expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [2] but found [1],
expected [false] but found [true]
at com.w2a.rough.TestNG.test2(TestNG.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="com.w2a.rough.TestNG" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Test -->
